artifacts/testdata/windows/init.cmd

```diff
@@ -1,2 +1,2 @@
 mklink "c:\users\link" "c:\Windows"
-start "artifacts\testdata\files\nc.exe -L -p 3889 -s 0.0.0.0"
\ No newline at end of file
+start cmd /c "artifacts\testdata\files\nc.exe -L -p 3889 -s 0.0.0.0"
```

Comment:
@scudette I cant see this line eecuting in appveyor console, should I add this somewhere else?

It looks like this mlink command is acutaly being executed from here https://github.com/Velocidex/velociraptor/blob/80cde4b12b840189e7eb512ca8e160a3489fee94/appveyor.yml
is init.cmd redundant?